app/src/androidTest/java/com/github/se/project/ui/profile/CreateTutorProfileTest.kt

```diff
@@ -35,10 +39,8 @@ class CreateTutorProfileTest {
 
   // Mocking navigation actions
   private val mockNavigationActions = Mockito.mock(NavigationActions::class.java)
-  private val mockViewModel =
-      Mockito.mock(ListProfilesViewModel::class.java).apply {
-        Mockito.`when`(currentProfile).thenReturn(MutableStateFlow<Profile?>(null))
-      }
```

Comment:
Moving the mock setup to the setup function is a good consistency change, as that is how other tests are done